packages/docusaurus-types/src/index.d.ts

```diff
@@ -416,6 +421,11 @@ export type LoadContext = {
   siteConfig: DocusaurusConfig;
   siteConfigPath: string;
   outDir: string;
+  /**
+   * Directory where all source translations for the current locale can be found
+   * in. Constructed with `i18n.path` + `i18n.currentLocale.path` (e.g. i18n/en)
+   */
+  i18nDir: string;
```

Comment:
move to i18n?

I would also rename it to something like `localeDir` or `localePath`?

The name does not make it very clear it contains the locale segment

We also need to make it clear what is a locale URL route path vs a locale fs dir path somehow

Reply:
👍 Let's rename it to `localizationDir`? Or `l10nDir`? `localeDir` doesn't sound like a good concept.

Reply:
ok for `localizationDir` 👍